modules/mail.js

```diff
@@ -1,19 +1,22 @@
 'use strict';
 const nodemailer = require('nodemailer');
 const config = require('config');
-const getSecret = require('../modules/get-secret');
+const AWS = require('aws-sdk');
+const Raven = require('raven');
 
-let mailConfig = {
-    user: getSecret('ses.auth.user'),
-    password: getSecret('ses.auth.password')
-};
-// create reusable transporter object using the default SMTP transport
+// Use local environment AWS credentials in dev mode.
+// Otherwise, EC2 instances already have IAM instance roles
+// with valid ses:SendRawEmail permissions
+if (process.env.NODE_ENV === 'development') {
```

Comment:
The docs make it sound like this isn't needed:

http://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-shared.html

"The SDK for JavaScript automatically searches the shared credentials file for credentials when loading"

"By default, the SDK checks the AWS_PROFILE environment variable to determine which profile to use. If the AWS_PROFILE variable is not set in your environment, the SDK uses the credentials for the [default] profile."

Does this work without these lines?

Reply:
It does!